src/core/server/opensearch/client/client_config.ts

```diff
@@ -55,6 +55,7 @@ export type OpenSearchClientConfig = Pick<
   | 'username'
   | 'password'
 > & {
+  // memoryCircuitBreaker?: OpenSearchConfig['memoryCircuitBreaker'] | ClientOptions['memoryCircuitBreaker']
```

Comment:
This isn't supposed to be commented out, right?

Reply:
This is commented on purpose because currently OSD is still use `elastic/elasticsearch`. This should be enabled once we start consuming `opensearch_project/opensearch`

Reply:
https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1342 just got merged, you should be able to test that out now 😄

Reply:
We may still need wait for the `v1.1.0`. Working on the new npm package publish.